--- ./src/main/native/unix_jni_linux.cc.old	2023-01-18 15:55:26.121269042 +0700
+++ ./src/main/native/unix_jni_linux.cc	2023-01-18 17:41:32.233319245 +0700
@@ -31,7 +31,7 @@
   // functionality which is not compatible with not the
   // SUSv3-conformant one which returns an error code; see DESCRIPTION
 // at strerror(3).
-#if !__GLIBC__ || (_POSIX_C_SOURCE >= 200112L && !_GNU_SOURCE)
+#if !__GLIBC__ && !__USE_GNU || (_POSIX_C_SOURCE >= 200112L && !_GNU_SOURCE)
   if (strerror_r(error_number, buf, sizeof buf) == -1) {
     return std::string("");
   } else {
